secator/cli.py

```
@@ -468,12 +471,12 @@
 
 
 @config.command('get')
-@click.option('--full', is_flag=True, help='Show full config (with defaults)')
+@click.option('--user/--full', is_flag=True, help='Show config (user/full)')
 @click.argument('key', required=False)
-def config_get(full, key=None):
+def config_get(user, key=None):
 	"""Get config value."""
 	if key is None:
-		partial = not full and CONFIG != default_config
+		partial = user and CONFIG != default_config
```

Ruff (0.11.9)

Yoda condition detected

Rewrite as `default_config != CONFIG`

(SIM300)